CODE_OF_CONDUCT.md

```diff
@@ -82,24 +81,23 @@ Examples of unacceptable behaviour by Turing Way community members at any projec
 
 Participants who are asked to stop any inappropriate behaviour are expected to comply immediately.
 This applies to all Turing Way community events and platforms, either online or in-person.
-If a participant engages in behaviour that violates this Code of Conduct, any member of the core development team may warn the offender, ask them to leave the event or platform (without refund), or impose any other appropriate sanctions (see the [enforcement manual](#4-enforcement-manual) for details).
+If a participant engages in behaviour that violates this Code of Conduct, any member of the core development team may warn the offender, ask them to leave the event or platform (without refund), or impose any other appropriate sanctions (see the enforcement manual for details).
```

Comment:
@malvikasharan I think this fixes the last build warning.

Unfortunately, I think this file gets included after sphinx/myst-parser constructs reference targets so this cross references is marked as target not found.

Reply:
Thank you so much Jim for catching the source of errors. I think I was not getting any new fresh perspective on what was failing and why.